ql/src/test/queries/clientpositive/file_with_delimiter.q

```diff
@@ -0,0 +1,19 @@
+create external table test(code string,name string)
+ROW FORMAT SERDE
+   'org.apache.hadoop.hive.serde2.lazy.LazySimpleSerDe'
+ WITH SERDEPROPERTIES (
+   'field.delim'='\t')
+ STORED AS INPUTFORMAT
+   'org.apache.hadoop.mapred.TextInputFormat'
+ OUTPUTFORMAT
+   'org.apache.hadoop.hive.ql.io.HiveIgnoreKeyTextOutputFormat'
+   location '${system:test.tmp.dir}/test'
+ TBLPROPERTIES (
+   'skip.header.line.count'='1',
+   'textinputformat.record.delimiter'='|');
+
+
+   LOAD DATA LOCAL INPATH '../../data/files/header_footer_table_4/0003.txt' INTO TABLE test;
+
+
+   Select count(*) from test;
```

Comment:
Could you please add select * to the output as well? Just to make sure we receive the expected output.

Reply:
Okay, I will add it. Thanks